types/csvrow/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "strictFunctionTypes" : false
```

Comment:
Need to set `"strictFunctionTypes" : true` here

Reply:
OK, no problem. Just so I understand though, could you explain why? Is it now standard to have strict function types? It seems to me the default behaviour before this option was introduced was that this was essentially false - is it the case that it's just best practise to have the strictest type checking possible unless there are specific reasons not to for specific checks?

Reply:
>is it the case that it's just best practise to have the strictest type checking possible unless there are specific reasons not to for specific checks?

Yes

Reply:
OK cool, thanks for clarifying.